mne/evoked.py

```diff
@@ -870,16 +873,15 @@ def combine_evoked(all_evoked, weights):
         if weights == 'nave':
             weights = naves / naves.sum()
         else:
-            weights = np.ones_like(naves)
+            weights = np.ones_like(naves) / len(naves)
     else:
-        weights = np.array(weights, float)
+        weights = np.squeeze(weights).astype(float)
```

Comment:
```suggestion
        weights = np.array(weights).astype(float)
```

I hate squeeze magic

Reply:
I don't see why we would need to use `squeeze` here anyway…?

Reply:
It's not really necessary. My thinking was that we are testing that ndim == 1, and I don't think the docstring explicitly says that weights must be a **1D** array, so I thought using squeeze was being generous to the user. I don't mind changing it back though